15-components-lab_v16/src/01-timeconverter.jsx

```diff
@@ -1,12 +1,57 @@
-/**
- * Lab 01: 
- *
- * Implement a time converter component.
- * Component should show 3 input fields labeled hours, minutes and seconds.
- * Typing a value in either field updates the other two with corresponding values.
- */
-
 import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Myinput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { 
```

Comment:
Why do you need 3 data values stored in state? I think you can easily reduce it to 1